### PR TITLE
Create Baseline

### PR DESCRIPTION
**Pull request by gkswns3708**
기초적인 Utils와 Baseline 코드가 작성되어 push합니다.

###  완료한 Task

- [x] Raw Data 경로의 File을 전부 Prepared Data의 경로에 Transform을 거쳐서 만들기
- [x] 기초적인 학습과 Matric 계산을 위한 코드 작성
- [x] PyTorch Framework 사용을 위한 Dataset, Dataloader 작성

### TODO

- [x] PrePared Data가 아닌 Raw Data만을 가져와서 Prepared Data로 Transform 하는 코드 작성
- [ ] 제대로된 Matric 작성을 위한 지식 습득 (Class Imbalance 문제가 있으므로 Focal Loss Implement, Confusion Matrix 작성 예정)
- [x] Local에서 GPU연산이 되도록 환경 설정
- [ ] Config들을 `Config.json`하나만으로도 조절할 수 있도록 작성할 예정
- [x] 배포 단계를 어떻게 진행할 것인지? (FastAPI를 생각하고 있음)
- [x]  Module화 된 코드를 작성하고 싶음. ipynb 작성 이후 Module화 예정.
- [ ]  자동화 Grid Search Training에 대한 구현 방법
- [ ]  Stratified하게 Fold가 진행되었는지 확인하는 코드(Class별 Visualization Code 작성 요망)
- [ ]  train.csv에 대한 고민

등 일단 생각나는대로 적어보았습니다!

### CODE REVIEW

**Comment by gkswns3708** [comment]
TODO를 보면서 일단 진행하되, TODO를 위한 PR도 하나 남겨놓아 계속해서 TODO가 갱신될 수 있도록 하면 좋을 듯 하다.
